### ISSUE TITLE
type of allowImageExifOrientation doesn't match the doc

### ISSUE THREAD

**Issue by imekachi** [pqina/filepond-plugin-image-exif-orientation#8]
# Problem
`<FilePond />` requires `allowImageExifOrientation` prop, which should be optional.

![image](https://user-images.githubusercontent.com/5445243/111874312-468bdb00-89c7-11eb-9d2d-1aad9add7b62.png)

# Description
According to this [doc](https://pqina.nl/filepond/docs/patterns/plugins/image-exif-orientation/), `allowImageExifOrientation = true` by default

Which mean `<FilePond />` shouldn't require the prop `allowImageExifOrientation`.

# Proposed solution
Make this prop optional

```diff
- allowImageExifOrientation: boolean;
+ allowImageExifOrientation?: boolean;
```

https://github.com/pqina/filepond-plugin-image-exif-orientation/blob/93f7eddcda98f445fe28a07ca164bcd71fac9e80/types/index.d.ts#L7

**Comment by rikschennink** [comment]
Thanks for reporting! Should be fixed in 1.0.11, just published

**Comment by imekachi** [comment]
@rikschennink Fixed, thank you. That was fast.
